platform/linux-generic/pktio/loop.c

```diff
@@ -233,7 +233,8 @@ static int loopback_recv(pktio_entry_t *pktio_entry, int index ODP_UNUSED,
 					odp_packet_free(pkt);
 
 					if (new_pkt == ODP_PACKET_INVALID) {
-						pktio_entry->s.stats.in_discards++;
+						odp_atomic_inc_u64(&pktio_entry->s
+								   .stats_extra.in_discards);
```

Comment:
The comment for `pktio_entry::stats_extra` could be improved if they nevertheless are used in pktio drivers.

Reply:
I'm thinking of just adding the word "also", i.e. "Statistics counters used also outside drivers". Good enough?

Reply:
OK to me.